flowtools/flowclr_sample_name/sample_simp.R

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/Rscript
+# Module for Galaxy
+# Adds sample information to a flowclustered file and binds 
+#
+#                  
+# Version 1
+# Emily Combe
+#
+#
+
+
+
+#-------------#
+# sample name #
+#-------------#
+
+
+library(flowCore)    
+
+sample_name <- function(input_file, sample_names){
+	flowtext <- list()
+	for(i in input_file){
+		flowtext[i] <- read.table(input_file[i], header = TRUE, sep = "\t")
+		flowtext[i] <- as.data.frame(flowtext[i])	
+	}
+	
+	sample_names <- as.character(sample_names)
+	
+	for(i in length(flowtext)){
+   		 flowtext[[i]]$sample <- rep(sample_names[i], nrow(flowtext[[i]]))
+    	}
+
+	if(length(flowtext) > 1){
+		flowtext <- rbindlist(flowtext)
+	} else {
+		flowtext <- flowtext[[1]]
+	}
+
+	return(flowtext)
```

Comment:
here you are just returning the list of data.frames, don't you need to write them into a file?